components/SharingWorkflow.integration.test.tsx

```
@@ -0,0 +1,542 @@
+    );
+
+    // Check that the main component loads
+    await waitFor(() => {
+      expect(screen.getAllByText("John Doe")[0]).toBeInTheDocument();
+    });
+
+    // Verify that the expired link mock is configured correctly
+    const sharingLinkMock = expiredLinkMocks.find(
+      (mock) =>
+        mock.request.query === GET_SHARING_LINKS_QUERY &&
+        "sharingLinks" in mock.result?.data &&
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)